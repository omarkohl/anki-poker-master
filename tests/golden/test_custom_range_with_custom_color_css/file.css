td.call {
    background-color: #4BE488;
}
td.fold {
    background-color: #D6D2D2;
}
td.my_special_range {
    background-color: red;
}
td.raise {
    background-color: #FF6A6A;
}
