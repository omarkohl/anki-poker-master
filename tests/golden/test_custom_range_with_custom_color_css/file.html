<script>td.call {
    background-color: #4BE488;
}
td.fold {
    background-color: #D6D2D2;
}
td.my_special_range {
    background-color: red;
}
td.raise {
    background-color: #FF6A6A;
}
</script><table class='legend'>
    <tr>
        <th class='row'>Call</th>
        <td class='call'>&nbsp;</td>
    </tr>
    <tr>
        <th class='row'>Fold</th>
        <td class='fold'>&nbsp;</td>
    </tr>
    <tr>
        <th class='row'>My special range</th>
        <td class='my_special_range'>&nbsp;</td>
    </tr>
    <tr>
        <th class='row'>Raise</th>
        <td class='raise'>&nbsp;</td>
    </tr>
</table>
